frontend/src/main/java/com/psephoi/mvc/manager/PollManagerImpl.java

```diff
@@ -0,0 +1,70 @@
+package com.psephoi.mvc.manager;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import com.psephoi.mvc.model.Poll;
+
+@Service
+public class PollManagerImpl implements PollManager{
+	
+	private final String restURL = "http://localhost:8080/";
+	
+	@Autowired
+	private RestTemplate restTemplate; 
+
+	
+	public List<Poll> list() {
+		String requestUri  = restURL + "poll/";
+		Poll[] response = restTemplate.getForObject(requestUri , Poll[].class);

```

Comment:
restTemplate.exchange is perferred.   If you update the Controller Endpoints on the provider to return a ResponseEntity rather than the POJO, be aware that it necessitate a change from getForObject() in this RestTemplate.